collections/hitech95/ngnix-mail.md

```diff
@@ -0,0 +1,37 @@
+## Ngnix Mail collection
+
+A collection for Nginx mail proxy
+ * Ngnix Mail core module log parsers
+ * Ngnix Mail auth module scenario bruteforce spam attempt
+
+It raccomanded to have the `crowdsecurity/nginx` collection installed!
+
+> Contribution by https://github.com/hitech95
+
+## Acquisition template
+
+Example acquisition for this collection :
+
+```yaml
+filenames:
+  - /var/log/nginx/*.log
+  - ./tests/nginx/nginx.log
+#this is not a syslog log, indicate which kind of logs it is
+labels:
+  type: nginx
+```
+
+If you are running a ngnix inside docker, like [mailu](https://mailu.io/):
```

Comment:
nginx*

Reply:
There is also a typo the collection configuration and doc filenames.

Reply:
Fixed, I really hate how nginx sounds in my native language.